server/api_authenticate.go

```diff
@@ -660,19 +661,44 @@ func (s *ApiServer) AuthenticateSteam(ctx context.Context, in *api.AuthenticateS
 	return session, nil
 }
 
+func (s *ApiServer) AuthenticateRefresh(ctx context.Context, in *api.AuthenticateRefreshRequest) (*api.Session, error) {
+	uid := ""
+	username := ""
+	var vars map[string]string
```

Comment:
Rather than read `vars` from the token allow it to be passed through in the `AuthenticateRefresh` request body, gives us extra flexibility to replace them if needed.